conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,298 @@
+      UTdTaDl0TzVLMWZsQUp6SnVKeTZaYlNtUUFscld1WjU4UHNSek1aUGc9PQ==
+    email_address: cortx@seagate.com # Optional 
+    mgmt_admin: cortxadmin
+    mgmt_secret: !!binary |
+      UlU1U0NrYklraS0ta3BOamN5X2J1VmUwRnJXTUFlb2JDdkItY1B3ZEE9PQ==
+      
+  s3:
+    auth_admin: admin
+    auth_secret: !!binary |
+      TVRGNDBQVnY3OXJ5bG4tLTQwSjNZQ0xQZVJiczBmUVNqcXFvTXdvNHc9PQ==
+    data:
+      endpoints:
+      - http://s3.seagate.com:80
+      - https://s3.seagate.com:443
+    iam:
+      endpoints:
+      - https://iam.seagate.com:9443
+    internal:
+      endpoints:
+      - http://s3.seagate.com:28049
+    io_max_units: 8
+    service_instances: 2
+    limits:   
+      num_services: 2                     
+    - service: rgw_s3
+      mem: 1Gi-2Gi
+      cpu: <Min>-<Max>
+    - service: rgw_admin
+      mem: 1Gi-2Gi
+      cpu: <Min>-<Max>
+
+  hare:
+    control:
+      endpoints:
+      - https://motr-hax-svc:22003
+    hax:
+      endpoints:
+      - tcp://hax.seagate.com:28000
+    limits:
+      num_services: 1
+    - name: hax
+      mem: 1Gi-2Gi
+      cpu: <Min>-<Max>
+      
+  motr:
+    client_instances: 1            # To be removed
+    interface_type: tcp            # To be removed (This is now part of URL)
+    interface_family: inet         # Optional: inet (default) | inet6
+    transport_type: libfab         # libfab | lnet (libfab==inet)
+    
+    ios:
+      endpoints:
+      - tcp://data1-node1:21001    # Format: <protocol>://<hostname_ip>:<port>
+                                   # N endpoints one for each Pod, first mentioned here
+                                   # N == Number of CVGs in the Pod
+                                   # Ports: 21001 - (21001+Number of CVGs)
+      - tcp://data2-node1:21001    # Same thing for the second pod
+      - tcp://data1-node2:21001    # of for Pods on the other nodes
+ 
+    confd:
+      endpoints:
+      - tcp://data1-node1:22002    # Format: <protocol>://<hostname_ip>:22002
+      - tcp://data1-node2:22002       
+      - tcp://data1-node3:22002    
+      
+    limits:                        # NOTE: This section is different from solution config
+      num_services: 1
+    - name: ios
+      mem: 1Gi-2Gi
+      cpu: <Min>-<Max>
+        
+    clients:                       # Motr Client Settings     
+    num_clients: 2                 # Number of Clients
+    - name: rgw_s3                 # rgw_s3|s3|cclient (Motr Clients)
+      num_instances: 1             # Client instances per node (pod)
+      endpoints:
+      - tcp://server1-node1:21001  # Format: <protocol>://<hostname_ip>:21001 
+      - tcp://server1-node2:21001  # Num endpoints will be same as num of client instances
+                                   # Port range would be 21001 - 21001+num_instances
+    - name: rgw_admin             
+      num_instances: 1           
+      endpoints:
+      - tcp://server1-node1:21001  
+      - tcp://server1-node2:21001  
+      
+    - name: cclient             
```

Comment:
If this is not cortx-s3server and just some other motr client, then I think `m0client` is better than `cclient` as we have client implemented in golang as well, technically motr clients can be written in languages other than C as well.

Reply:
not done, will be doing it in the second iteration of changes

Reply:
done